packages/atlas/src/components/NftTileViewer/NftTileViewer.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router'
+
+import { useNft } from '@/api/hooks'
+import { AllNftFieldsFragment } from '@/api/queries'
+import { absoluteRoutes } from '@/config/routes'
+import { useBlockTimeEstimation } from '@/hooks/useBlockTimeEstimation'
+import { useAsset } from '@/providers/assets'
+
+import { NftTile, NftTileProps } from '../NftTile'
+
+type NftTileViewerProps = {
+  nftId?: string
+}
+
+export const NftTileViewer: React.FC<NftTileViewerProps> = ({ nftId }) => {
+  const { nft, loading } = useNft(nftId || '')
```

Comment:
We should probably add `skip` if `nftId` is not set

Reply:
We already have this inside the `useNft` hook